pulsar-broker/src/test/java/org/apache/pulsar/broker/systopic/SystemTopicClientTest.java

```diff
@@ -22,30 +22,55 @@
 import static org.apache.pulsar.common.naming.TopicName.TRANSACTION_COORDINATOR_LOG;
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
+import org.apache.pulsar.broker.namespace.NamespaceService;
+import org.apache.pulsar.broker.service.BrokerService;
+import org.apache.pulsar.broker.service.BrokerTestBase;
+import org.apache.pulsar.common.naming.NamespaceName;
 import org.apache.pulsar.common.naming.TopicName;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-public class SystemTopicClientTest {
+public class SystemTopicClientTest extends BrokerTestBase {
+
+    @Override
+    @BeforeMethod(alwaysRun = true)
+    protected void setup() throws Exception {
+        super.baseSetup();
+    }
+
+    @Override
+    @AfterMethod(alwaysRun = true)
+    protected void cleanup() throws Exception {
+        super.internalCleanup();
+    }
 
     @Test
     public void testIsSystemTopic() {
```

Comment:
It's better to move this test to `BrokerServiceTest`.

Reply:
ok, yes, right.